lib/workable/client.rb

```diff
@@ -273,6 +297,12 @@ def do_request(url, type, &_block)
 
     # parse the api response
     def parse!(response)
+      rate_limit_remaining_header = response['X-Rate-Limit-Remaining']
+      # Fail faster so that the API doesn't penalise us with a larger reset date as often.
+      if rate_limit_remaining_header.present? && rate_limit_remaining_header.to_i <= 1
```

Comment:
rate_limit_remaining_header.to_i, is it possible to be negative values? or is it checking for 0 with a safeguard in case it's negative?